servo.go

```diff
@@ -0,0 +1,138 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"math"
+	"time"
+
+	"github.com/felixge/pidctrl"
```

Comment:
I don't see this in Gopkg.lock. You'll need to run `dep ensure` to vendor this dependency and add those files to the PR.